lib/rubocop/cop/mixin/negative_conditional.rb

```diff
@@ -6,15 +6,20 @@ module Cop
     # Some common code shared between FavorUnlessOverNegatedIf and
     # FavorUntilOverNegatedWhile.
     module NegativeConditional
+      def self.included(mod)
```

Comment:
Since this is a mixin it doesn't have access to `def_node_matcher` which is used as an easy way to test the conditional for a double negation. Without it we would need to drill into the second level of the condition [here](https://github.com/bbatsov/rubocop/pull/3146#discussion-diff-64073654R21) which seems like a worse option to me